Core/HLE/__sceAudio.cpp

```diff
@@ -199,16 +191,14 @@ void __AudioDoState(PointerWrap &p) {
 
 	p.Do(mixFrequency);
 
-	{	
-		//block until a lock is achieved. Not a good idea at all, but
-		//can't think of a better one...
-		__blockForAudioQueueLock();
-
+	if (s >= 2) {
+		resampler.DoState(p);
+	} else {
+		// Only to preserve the previous file format. Might cause a slight audio glitch on upgrades?
```

Comment:
Well, I think the important thing is that we're clearing it, so it doesn't fuzz out.

-[Unknown]